### ISSUE TITLE
How to produce? ...and possibly a screw or connection terminal as an alternative for AT cables

### ISSUE THREAD

**Issue by CKoewing** [hkzlab/ATnoX#1]
Hi Fabio,

I think your ATnoX adapter is really great, just what I've been looking for for probably two weeks now. :-D

The only problem I have is that I am neither an electrician nor a PCB designer and I don't know how to produce a PCB from your Github repository at e.g. JLCPCB, PCBway, etc. Also, I can solder, but it stops with SMD components.

Can you tell me what I can do?

How about a revision 3.3 converted to through-hole components? ;-P

By the way, it would be great if instead of the 24-pin ATX-A-Like connector, I could alternatively solder on screw or connection terminals, so that I could simply cut off old AT cables, fit them with wire sleeves and screw them on.

Thanks and best regards,
Chris

**Comment by hkzlab** [comment]
To produce the board you simply have to provide gerber files to the PCB Fab.
You can find already bundled gerbers inside the repository, here: https://github.com/hkzlab/ATnoX/tree/master/ATnoX/gerbers
Just use the latest zip and send it to the fab: usually the process is semi-automated via web interface E.g. https://cart.jlcpcb.com/quote

This project actually started as through-hole, but I did not like the size of the result. I currently don't have plans to do it, but if I ever get back to it and make a through hole version, it won't be in this repo, and will make a new project.

**Comment by CKoewing** [comment]
Hi Fabio,

means I can't get around SMD soldering, right?

I had already uploaded the Gerber data to JLCPCB, but then I didn't really know which ticks to set for which options. Do you have any recommendations? Dos and don'ts?

Also JPCPCB offers to mount the components on a PCB, but this is currently only possible on one side. 
And if I have seen this correctly, components are installed on both sides of your PCB. Right?

So what about a PCB revision 3.3 with the components installed on at least one side only, so that users like me could have the SMD components installed at JLCPCB?

This way you would also address a somewhat wider range of people with your great project. :-D

Sorry if I'm annoying you, I know computer hardware and software development, but I haven't had much to do with PCB's, except for very simple things using through-hole components.

Thank you very much in any case. :-)

**Comment by hkzlab** [comment]
Exactly, in this case you can't get around it. None of the components on this board are fine pitched though, or require an especially steady hand.
JLCPCB offers the mounting of components at the fab, but it doesn't come "free": you need to provide files to match components to the footprints. Also, placing components on only one side of the board requires an extensive redesign, not to mention a significant increase in size of the board. I'm concentrated on other (unpublished, for now) projects in the very limited spare time I had these last months.

It would actually be a nice exercise for someone interested in learning KiCad: taking a relatively simple project like this and start rearranging components :wink: Not to mention I have not gotten around to add the BOM to the README yet, so you will have to open the project with kicad to get the exact values of the components for now.

The values autodetected by JLCPCB order module are fine. Just pay attention to layer ordering. See screenshot
![Screenshot from 2021-11-08 19-17-39](https://user-images.githubusercontent.com/603354/140796356-9c074125-4d45-4057-9a8b-4a6b1a8ad595.png)

**Comment by CKoewing** [comment]
Ah, okay, so it is really a 4 layer PCB. Was thinking, that this is a fault of JLCPCB.

**Comment by hkzlab** [comment]
Hello Chris! 
Gave a very quick glance! Good job!
I'll have a more in-depth look as soon as I have some free time (a bit hectic), but I greatly appreciate whomever takes projects and adapts them for their own use :)

**Comment by chris-kwng** [comment]
Now including a useful readme. 🙂
